daemon/cmd/daemon_main.go

```diff
@@ -936,6 +936,9 @@ func InitGlobalFlags(logger *slog.Logger, cmd *cobra.Command, vp *viper.Viper) {
 	flags.Float64(option.ConnectivityProbeFrequencyRatio, defaults.ConnectivityProbeFrequencyRatio, "Ratio of the connectivity probe frequency vs resource usage, a float in [0, 1]. 0 will give more frequent probing, 1 will give less frequent probing. Probing frequency is dynamically adjusted based on the cluster size.")
 	option.BindEnv(vp, option.ConnectivityProbeFrequencyRatio)
 
+	flags.Float64(option.BPFPolicyMapPressureMetricsThreshold, defaults.BPFPolicyMapPressureMetricsThreshold, "Threshold for triggering policy map pressure metrics, a float in [0, 1]")
```

Comment:
AFAICT, this is only used in `pkg/endpointmanager`. Thus, please add it to `pkg/endpointmanager/config.go` instead.